Kickstarter-iOS/Views/Cells/SettingsNotificationCell.swift

```diff
@@ -38,6 +38,7 @@ final class SettingsNotificationCell: UITableViewCell, NibLoading, ValueCell {
 
     _ = titleLabel
       |> UILabel.lens.text .~ cellValue.cellType.title
+      |> \.accessibilityElementsHidden .~ cellValue.cellType.accessibilityElementsHidden
```

Comment:
Is there a reason we're using `accessibilityElementsHidden` and not `isAccessibilityElement`? I'm not sure that `titleLabel` can have any subviews?

Reply:
When `isAccessibilityElement` is set to `false` for the notification cells w/  email/push icons and `true` for `Manage project notifications` button, VoiceOver as expected, would avoid focusing on the notification cell titles, however, for `Manage project notifications` button  VoiceOver was reading the button title and number of notifications separately. If the user tapped over the cell VoiceOver was only reading the number of notifications, the user would have to tap over the title for it to read.  With `accessibilityElementsHidden` VoiceOver avoids focusing on reading the titles of the notification cells w/ email/push icons while still reading the `Manage Project notifications` title and number of notifications together.

Reply:
So the only way I was able to have this work properly was to use your original approach + fix the order of the buttons like so:

```
func configureWith(value cellValue: SettingsNotificationCellValue) {
    self.notificationType = cellValue.cellType

    viewModel.inputs.configure(with: cellValue)

    let accessibilityElementsHidden = cellValue.cellType.accessibilityElementsHidden

    _ = self
      |> \.accessibilityTraits .~ cellValue.cellType.accessibilityTraits

    _ = self.stackView
      |> \.accessibilityElements .~ (
        accessibilityElementsHidden ? [self.emailNotificationsButton, self.pushNotificationsButton] : nil
    )

    _ = self.titleLabel
      |> UILabel.lens.text .~ cellValue.cellType.title
      |> \.accessibilityElementsHidden .~ accessibilityElementsHidden

    _ = self.arrowImageView
      |> UIImageView.lens.isHidden .~ cellValue.cellType.shouldHideArrowView
      |> UIImageView.lens.tintColor .~ .ksr_dark_grey_400

    _ = self.projectCountLabel
      |> UILabel.lens.isHidden .~ cellValue.cellType.projectCountLabelHidden
      |> \.accessibilityElementsHidden .~ accessibilityElementsHidden
  }
```

please notice that:
* I've added `self.` to reference `IBOutlet`s consistently
* Also set `accessibilityElementsHidden` on the `projectCountLabel` (it worked without it but might be confusing as to why since we don't won't either of the labels to be accessible and let the cell take over instead)